app/static/src/app/components/VerticalCollapse.vue

```diff
@@ -2,34 +2,37 @@
   <a data-bs-toggle="collapse"
      :href="`#${collapseId}`"
      role="button"
-     aria-expanded="true"
+     :aria-expanded="!collapsed"
```

Comment:
there's a comment in the previous review here, but ideally if we can persist the collapsed/expanded state for shared sections when changing tabs on the rhs